source/webhook-trigger.py

```diff
@@ -97,26 +110,17 @@ def verify_environment(variables):
     for variable in variables:
         value = os.getenv(variable)
         if value is None:
-            logging.error(f"Missing variable [{variable}] - {variables[variable]}")
+            message = exception_message(f"Missing variable [{variable}] - {variables[variable]}", None)
+            logging.error(message)
             sys.exit(1)
 
 
-def send_slack_error_message(event, status_code):
-    values = extract_payload(event)
-    if status_code is not None:
-        values["status_code"] = status_code
-
-    slack_webhook_url = os.getenv("SLACK_WEBHOOK_URL")
-
-    loader = jinja2.FileSystemLoader("templates")
-    env = jinja2.Environment(loader=loader)
-    env.filters["jsonify"] = json.dumps
+if sys.version_info < MIN_PYTHON:
+    error_message = "Found Python " + str(sys.version_info) + " but Python %s.%s or later is required. " \
+                                                              "Unable to find towel. Panicking.\n" % MIN_PYTHON
```

Comment:
As discussed we should probably keep the error messages concise

Reply:
Have changed to `"Found Python " + str(sys.version_info) + " but need Python %s.%s or later." % MIN_PYTHON`